src/hierarchical_deterministic/cap26/cap26_path/cap26_path_uniffi_fn.rs

```diff
@@ -0,0 +1,84 @@
+use crate::prelude::*;
+
+#[uniffi::export]
+pub fn new_cap26_path_from_string(
+    cap26_path_string: String,
+) -> Result<CAP26Path, CommonError> {
+    CAP26Path::from_str(&cap26_path_string)
+}
+
+#[uniffi::export]
+pub fn default_get_id_path() -> GetIDPath {
+    GetIDPath::default()
+}
+
+#[uniffi::export]
+pub fn cap26_path_to_string(path: &CAP26Path) -> String {
+    path.to_string()
+}
+
+#[uniffi::export]
+pub fn new_account_path_sample() -> AccountPath {
+    AccountPath::sample()
+}
+
+#[uniffi::export]
+pub fn new_account_path_sample_other() -> AccountPath {
+    AccountPath::sample_other()
+}
+
+#[uniffi::export]
+pub fn new_identity_path_sample() -> IdentityPath {
+    IdentityPath::sample()
+}
+
+#[uniffi::export]
+pub fn new_identity_path_sample_other() -> IdentityPath {
+    IdentityPath::sample_other()
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_account_path() {
+        let path = CAP26Path::from(new_account_path_sample());
+        assert_eq!(
+            path,
+            new_cap26_path_from_string(cap26_path_to_string(&path)).unwrap()
+        );
+
+        let path_other = CAP26Path::from(new_account_path_sample_other());
```

Comment:
`let path_other = CAP26Path::from(AccountPath::sample_other());` ?

Reply:
The reason for this is to also invoke the sample methods and count them in to the code coverage.

Reply:
right, you can add what I usually do:
 (see `public_key_hash_uniffi_fn.rs`)
```rust
    #[test]
    fn hash_of_samples() {
        assert_eq!(
            HashSet::<SUT>::from_iter([
                new_public_key_hash_sample(),
                new_public_key_hash_sample_other(),
                // duplicates should get removed
                new_public_key_hash_sample(),
                new_public_key_hash_sample_other(),
            ])
            .len(),
            2
        );
    }
```